pkg/sysinfo/sysinfo.go

```diff
@@ -25,17 +25,15 @@ func New(quiet bool) *SysInfo {
 			logrus.Warnf("%s", err)
 		}
 	} else {
-		_, err1 := ioutil.ReadFile(path.Join(cgroupMemoryMountpoint, "memory.limit_in_bytes"))
-		_, err2 := ioutil.ReadFile(path.Join(cgroupMemoryMountpoint, "memory.soft_limit_in_bytes"))
-		sysInfo.MemoryLimit = err1 == nil && err2 == nil
+		sysInfo.MemoryLimit = err == nil
```

Comment:
Just for clarification: With this change, we assume that `limit_in_bytes` and `soft_limit_in_bytes` is there because `cgroup/memory` exists?